.travis.yml

```diff
@@ -1,8 +1,19 @@
+sudo: required
+
 language: go
 
+services:
+  - docker
+
 go:
   - master
 
+before_install:
+- docker pull lleseigneur/bonita-postgres
+- docker run -d --name postgres -p 127.0.0.1:15432:5432 lleseigneur/bonita-postgres
```

Comment:
Isn't there a `docker stop ...` missing after all?

Reply:
you're right
in fact problem is to get IP or hostname of this container to run test on it